pkg/network/master/egressip.go

```diff
@@ -170,6 +177,21 @@ func (eim *egressIPManager) check(retrying bool) bool {
 			continue
 		}
 
+		nn, err := eim.nodeInformer.Lister().Get(node.name)
+		if err != nil {
+			return true, err
```

Comment:
I would just warn on the error here and treat the node as up; if it's disappeared we will deal with that shortly.